### ISSUE TITLE
Hüllenintegritätsanzeige für alle Fenster in der c-base

### ISSUE THREAD

**Issue by uwekamper** [c-base/meta#170]
Holzfenster, kann man direkt Reedkontakte reinschrauben.

Reedkontakte im Baumarkt oder bei Conrad besorgen.

Kabel mit dickem Querschnitt.

Hoehere Spannung, 25V? 48V statt 12V?

**Comment by ij0n** [comment]
vielleicht wollen wir das per MQTT übertragen? mit so kleinen ESP-Wlan-Chips?

**Comment by berlincount** [comment]
ich find das 2.4 GhZ auf der base schon ziemlich überlastet, ich würde da nicht sicherheitsrelevante funktionalität dran binden wollen ... kabel ftw.

**Comment by XenGi** [comment]
MQTT++
ESPs sind billig und einfach zu benutzen. Sollten wir einfach mal wieder ne Schippe bei Ali bestellen.

Gibt es den nda ne billige Alternative mit Ethernet? Vielleicht http://vocore.io/ ?

**Comment by berlincount** [comment]
Bitte KEINE Dependency auf "Working WiFi" für sicherheitsrelevante Dinge einführen.

Ich mag sowohl ESPs als auch MQTT, aber in diesem Fall sollte MQTT wirklich via crewnetz beschickt werden.

Oder MINDESTENS ein "wenn MQTT in der letzten Minute nicht mit 'alles zu' aktualisiert wurde, wird von offen ausgegangen", was dann aber potentiell viele False Positives erzeugen kann. Wenn die Station laut Schalter an der Tür auf "Shutdown" steht sollte m.E. in dem Fall eine Mail an cey@ ausgelöst werden.

Ansonsten kann man Reed-Kontakte als Schliesser alle seriell an einen GPIO hängen und hat zumindest einen binären Status für ganze Fensterfronten etc.

**Comment by berlincount** [comment]
Sack ESPs: diskutieren, berechnen, beantragen, Wohlwollen beim Cash erwarten. Das 2.4 GhZ Band ist eh schon hinüber, is dann auch egal.

**Comment by XenGi** [comment]
Also Ethernet statt WLAN wär ich auch ein Fan von. Fehlt halt ein Board das das kann. Im Zweifel ein Arduino oder wie gesagt der vocore.

Für so IoT foo haben wir übrigens das c-base-botnet. Sollte doch irgendwas ins WLAN sollen.

**Comment by berlincount** [comment]
einen VoCore hab ich auch noch. Jo.

Dass wir c-base-botnet haben hilft uns überhaupt nicht, wenn die Frequenzen platt sind.

**Comment by XenGi** [comment]
> Dass wir c-base-botnet haben hilft uns überhaupt nicht, wenn die Frequenzen platt sind.

Ich meine das auch eher als seperates Netz und nicht als seperates WLAN. Den ganzen IoT Krempel inkl. "intelligenter" Lampen etc will man vlt nicht ungebdingt im normalen Netz haben. Gleichzeitig will man aber auch Zugriff auf den MQTT Server haben der ja bekanntlich im Crewnetz ist.
Ein Netz für komische Dinge die miteinander reden wäre also sinnvoll. Ob die das nun über WLAN machen oder LAN ist dabei erstmal egal. LAN ist natürlich immer zu bevorzugen.

Damit wäre aber das normale Crenetz von MQTT Traffic und sonstigem seltsamen Kram befreit und man kann z.b. Dinge besser debuggen sollte man mal was scannen müssen.

**Comment by uwekamper** [comment]
Boah ey! Nerds! Echt ey!

Mein ursprünglicher Plan war, die gleiche Schaltung wie im c-lab zu bauen. Komplett ohne CPU irgendwo. Die Schaltung besteht aus mehreren Reed-Schaltern, langen Kabeln, einem Netzteil und LEDs. Ist sehr einfach. Geht nur selten kaputt. Wenn es doch kaputt geht, sieht man es sofort.

Wenn das funktioniert, können wir immer noch irgendeine MCU dran hängen, die den aktuellen Status ins Netz schickt.

**Comment by XenGi** [comment]
Ja dann mach halt. Zwingt dich ja niemand auf das Gehubel einzugeben. :)

**Comment by uwekamper** [comment]
Die Reed-Schalter sind an den Fenstern montiert. Kabel müssen noch verbunden werden.

**Comment by ij0n** [comment]
Muss immer noch fertig gemacht werden. 

Status: Mainhall hat an allen Fenstern Reed Schalter, aber die haben zwar Kabel sind aber nicht angeschlossen oder zusammen geführt. 

Seminar Raum braucht noch Schalter

**Comment by ij0n** [comment]
weiterhin offen - ESPHome nehmen und das alles über homeassistant routen? wäre das nicht sehr sehr einfach?

**Comment by uwekamper** [comment]
NEIN! zu ESPHome und homeassistant - das System muss komplett Analog mit nichts weiter als einem Netzteil, den Türschaltern und LED-Lichtern funktionieren (ohne Intelligenz) – genau so wie im c-lab.
